tests/baselines/reference/typeArgInference2.errors.txt

```diff
@@ -16,7 +16,7 @@ tests/cases/compiler/typeArgInference2.ts(12,10): error TS2453: The type argumen
     var z4 = foo({ name: "abc" });        // { name: string }
     var z5 = foo({ name: "abc", a: 5 });  // { name: string; a: number }
     var z6 = foo({ name: "abc", a: 5 }, { name: "def", b: 5 });  // error
-             ~~~
+                                                       ~~~~

```

Comment:
I suppose I have some reservations about this, but what would fix this is to change the topmost message to "The type argument for type parameter 'T' cannot be inferred from the usage **of 'foo'**."